projects/isy-angular-widgets/src/lib/wizard/components/wizard/wizard.component.ts

```diff
@@ -134,58 +135,70 @@ export class WizardComponent implements OnInit, AfterContentInit {
   /**
    * Stores the items of the wizard
    */
-  wizardItems: MenuItem[] = [];
+  items: MenuItem[] = [];
 
   /**
    * Fired on initialization
    */
   ngOnInit(): void {
-    this.stepperIndexChange.emit(this.index);
+    this.indexChange.subscribe(() => {
+      this.stepperIndexChange.emit(this.index);
+    });
+    this.indexChange.emit(this.index);
   }
 
   /**
    * Fired after content initialization
    */
   ngAfterContentInit(): void {
-    this.wizardItems = this.content.map((item) => {
+    this.items = this.content.map((item) => {
       return {
         label: item.isyWizardDirective
       };
     });
   }
 
   /**
-   * Moves the stepper to the next or previous position
-   * @param next used for forward/backward navigation
+   * Fired on changes
+   * @param changes Includes all DOM changes
    */
-  move(next: boolean): void {
-    this.stepper.move(next);
-    this.stepperIndexChange.emit(this.stepper.index);
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.isVisible && this.isVisible) {
+      if (changes.isVisible.currentValue === false) {
+        this.resetWizard();
+      }
+    }
```

Comment:
A little hard to read, can't this be simplified?

Reply:
Done